modules.rb

```diff
@@ -0,0 +1,6 @@
+module  All
+  def offer_high_five
+    "High five!"
+  end
+
+end
```

Comment:
Usually for modules that are going to be included in classes, we use "-able" names, so here it'd be HighFiveable. Module names should be related to the methods that are defined in it.